espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/KReflectionUtils.kt

```diff
@@ -0,0 +1,67 @@
+package io.appium.espressoserver.lib.helpers
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException
+import java.lang.IllegalArgumentException
+import java.lang.reflect.InvocationTargetException
+import kotlin.reflect.KClass
+import kotlin.reflect.KFunction
+import kotlin.reflect.full.createType
+import kotlin.reflect.full.functions
+import kotlin.reflect.full.memberFunctions
+import kotlin.reflect.jvm.javaType
+
+object KReflectionUtils {
```

Comment:
what is `K`? can we have a better class name?

Reply:
Kotlin. It follows the convention they use to distinguish Kotlin from Java (KClass, KType, etc...)